app/assets/javascripts/application_legacy.js

```diff
@@ -18,6 +18,7 @@
 //= require_self
 //= require big
 //= require units-converter
+//= require validation-utils
```

Comment:
I think the AI added some unused methods for these validation-utils (I added review comments for where I seem to have found dead code) and also the solution it provided supersedes some old logic, which it also failed to remove (e.g. the `%span.numeric-step-error` in the group order form template no longer seems to be required).

Instead of reviewing the AI code in depth (which seems cumbersome, as I think there's quite some AI slop), here's my quick go on this:

https://github.com/foodcoops/foodsoft/pull/1232/files#diff-ade87a685fd4a0e008033752780385cce4bbfff5d1355fab4b35cafffc2b3080R44